server/utils/pagination/pagination.ts

```diff
@@ -0,0 +1,95 @@
+import { Page } from '../../models/pagination'
+
+interface MojPaginationArgs {
+  classes?: string
+  items: (
+    | {
+        type: 'dots'
+      }
+    | {
+        type: 'pageNumber'
+        selected: boolean
+        text: number
+        href: string
+      }
+  )[]
+  previous?: {
+    text: string
+    href: string
+  }
+  next?: {
+    text: string
+    href: string
+  }
+  results?: {
+    from: number
+    to: number
+    count: number
+  }
+}
+
+export default class Pagination {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(private readonly page: Page<any>) {}
+
+  private constructPageItems(pageNumberRange: number[], chosenPageNumber: number): MojPaginationArgs['items'] {
```

Comment:
what does this type mean? `MojPaginationArgs['items']`

Reply:
MojPaginationArgs is an interface from Moj design for pagination.

We don't have @types defined for MOJ design - I see we do the same for other MOJ components in ViewUtils.